src/api/spec/models/workflow/step/rebuild_package_step_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Workflow::Step::RebuildPackage, vcr: true do
+  let!(:user) { create(:confirmed_user, :with_home, login: 'Iggy') }
+  let(:token) { create(:workflow_token, user: user) }
```

Comment:
I think it's not true. `subject` will be lazily evaluated on the `context` and it will use the `token` defined in the `context`.

Reply:
With _`token` is global_, I meant that it's defined at the top of the `describe` block and it's unclear why, since only the first example is using this. If you want to know what exactly is `token`, you need to look at the top of the file instead of just looking at the context itself. Anyway, it's not a blocker, but it's not a good practice. I know that we do this in many specs, but it's not right.